src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -75,6 +81,22 @@ def _create_ui_components(self):
                 schema=element,
             )
 
+    def _create_integration_feature(self):
+        # sentry apps must have at least one feature
+        # defaults to 'integrations-api'
+        try:
+            IntegrationFeature.objects.create(
+                sentry_app=self.sentry_app,
+            )
+        except IntegrityError as e:
+            logger.info(
```

Comment:
You can just use `self.log(extra={...})` here (https://github.com/getsentry/sentry/blob/master/src/sentry/mediators/mediator.py#L183).

Each mediator already has a logger and this function will add a bunch of other data, if available. It also makes sure the logger names are consistent.